homeworks/oleksandr.khodakivskyi_Moo0oony/7-array-loops/arrays.js

```diff
@@ -0,0 +1,120 @@
+const arrayOne = [3, 0, -5, 1, 44, -12, 3, 0, 0, 1, 2, -3, -3, 2, 1, 4, -2, -3, -1];
+const arrayTwo = [-1, -8, -2];
+const arrayThree = [1, 7, 3];
+const arrayFour = [1, undefined, 3, 5, -3];
+const arrayFive = [1, NaN, 3, 5, -3];
+
+function maxNumberOfArr(array) {
+    let maxNumber = array[0];
+    for (let i = 0; i < array.length; i++) {
+        if (array[i] > maxNumber) {
+            maxNumber = array[i];
+        }
+    }
+    return maxNumber;
+}
+
+console.log(maxNumberOfArr(arrayOne)); // 44
+console.log(maxNumberOfArr(arrayTwo)); // -1
+console.log(maxNumberOfArr(arrayThree)); // 7
+console.log(maxNumberOfArr(arrayFour)); // 5
+console.log(maxNumberOfArr(arrayFive)); // 5
+
+function minNumberOfArr(array) {
+    let minNumber = array[0];
+    for (let i = 0; i < array.length; i++) {
+        if (array[i] < minNumber) {
+            minNumber = array[i];
+        }
+    }
+    return minNumber;
+}
+
+console.log(minNumberOfArr(arrayOne)); // -12
+console.log(minNumberOfArr(arrayTwo)); // -8
+console.log(minNumberOfArr(arrayThree)); // 1
+console.log(minNumberOfArr(arrayFour)); // -3
+console.log(minNumberOfArr(arrayFive)); // -3
+
+function sumNumbersOfArr(array) {
+    let sum = 0;
+    for (let i = 0; i < array.length; i++) {
+        if (!Number.isNaN(array[i]) && array[i] !== undefined) {
+            sum = array[i] + sum;
+        }
+    }
+    return sum;
+}
+
+console.log(sumNumbersOfArr(arrayOne)); // 32
+console.log(sumNumbersOfArr(arrayTwo)); // -11
+console.log(sumNumbersOfArr(arrayThree)); // 11
+console.log(sumNumbersOfArr(arrayFour)); // 6
+console.log(sumNumbersOfArr(arrayFive)); // 6
+
+function onlyNegativeNumbers(array) {
+    const negativeArray = [];
```

Comment:
Hey, @Moo0oony, please use native array method for this and next fuction, as stated in the task description
> 2.) За допомогою методів масивів реалізуйте наступні функції:
> - onlyNegativeNumbers() - має повернути відфільтрований масив лише негативних чисел
> - onlyPositiveNumbers() - має повернути відфільтрований масив лише позитивних чисел

Reply:
Hi @vladyslav-yermolin-moc! Oh, haven't noticed this condition. Thanks :)